modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotRestoreProcess.java

```diff
@@ -86,6 +87,12 @@
     /** Reject operation message. */
     private static final String OP_REJECT_MSG = "Cache group restore operation was rejected. ";
 
+    /** Snapshot restore operation finish message. */
+    private static final String OP_FINISHED_MSG = "Successfully restored cache group(s) from the snapshot [reqId=%s]";
```

Comment:
Let's use the right word order Subject + Verb + Object: `Cache groups have been successfully restored from snapshot`.
Do we need a request Id? I'd rather use the cache group names or remove it at all.

Reply:
These constants were introduced in order to use the same message in logs and events. 
This is how the logging looks now, I don't think there is a problem with changing this wording in the log. 
But I think we must specify the request ID so that we can find the rest of the events/log-messages/etc related to this one.